package.json

```diff
@@ -56,10 +57,9 @@
     "ts-node": "^7.0.1",
     "tslint": "^5.11.0",
     "tslint-config-groupby": "^1.0.0",
-    "typedoc": "^0.13.0",
-    "typedoc-plugin-monorepo": "github:groupby/typedoc-plugin-monorepo",
+    "typedoc": "^0.14.2",
     "typedoc-plugin-nojekyll": "^1.0.1",
-    "typescript": "^3.1.3",
+    "typescript": "^3.4.5",
```

Comment:
To confirm: Updating a dependency busts CircleCI's node dependency cache (ie. the `build_*` tests are passing using _this_ version of TypeScript)?

Reply:
Yes, CircleCI's cache is based on the hash of the `yarn.lock` file. The dependency changes, the hash changes, and the cache for the new hash doesn't exist. Plus, we're only caching yarn's global cache (`~/.cache/yarn`) and not the `node_modules`, so even if it were cached, the new version would get pulled in anyway.

Reply:
Noted, thanks for clarifying. Marking this as resolved.